.husky/pre-commit

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
```

Comment:
While it's acceptable, what we need is, a combination of [lint-staged](https://github.com/okonet/lint-staged) + [husky](https://typicode.github.io/husky/#/) for performance & efficiency concerns. Give this article - "[How to Set Up Git hooks with husky and lint-staged](https://www.joshuacolvin.net/git-hooks-with-husky-and-lint-staged/)" a thorough read to understand why we need the aforementioned combination.

That said, we'll get back to cleaning it up once you've taken care of the other spam as well.